lib/matplotlib/backend_tools.py

```diff
@@ -589,6 +595,43 @@ class SaveFigureBase(ToolBase):
     default_keymap = rcParams['keymap.save']
 
 
+class ToolSaveFigure(ToolBase):
+    """Saves the figure"""
+
+    description = 'Save the figure'
+    image = 'filesave.png'
+    default_keymap = rcParams['keymap.save']
+
+    def get_filechooser(self):
+        fc = self.figure.canvas.backend.FileChooserDialog(
```

Comment:
I added it to the canvas in the commit before this... the reason for this, we will always have a canvas we won't always have a ``FigureManager``, thus it simplifies the code if canvas knows the ``backend`` it belongs to as well.